test/LinearSolvers/poisson.jl

```diff
@@ -131,28 +130,29 @@ let
 
   polynomialorder = 4
   base_num_elem = 4
+  tol = 1e-9
```

Comment:
Delete this in favor of default input?

Reply:
Cannot do that because `tol` is also used outside of `run` in calculating `isapprox` relative tolerance. Thanks for taking a look at this though, I think I found a better way to deal with `tol`.